recipe/ingredients.py

```diff
@@ -457,13 +475,9 @@ class Dimension(Ingredient):
     def __init__(self, expression, **kwargs):
         super(Dimension, self).__init__(**kwargs)
 
-        # An optional exprssion to use instead of the value expression
```

Comment:
order_by_expression used to be treated separately. Now it is handled like any other extra expression ("role") that gets passed into a dimension.